main.tf

```diff
@@ -55,32 +55,32 @@ module "lambda_cloudwatch_trigger" {
 }
 
 module "lambda_s3_trigger" {
-  source = "app.terraform.io/bankrate/lambda-s3-trigger/aws"
+  source  = "app.terraform.io/bankrate/lambda-s3-trigger/aws"
   version = "~> 1.0.0"
 
   # Enablement
-  enable              = var.enable && lookup(var.architecture, "s3_trigger", false)
+  enable = var.enable && lookup(var.architecture, "s3_trigger", false)
 
-  bucket_name           = var.bucket_id
+  bucket_name         = var.bucket_id
   lambda_function_arn = module.lambda.arn
 }
 
-module "lambda_ddb_trigger" {
-  source  = "app.terraform.io/bankrate/lambda-event-source/aws"
+module "lambda_event_source" {
+  source  = "app.terraform.io/RVStandard/lambda-event-source/aws"
```

Comment:
Module "lambda_event_source" cannot be found when running a `terraform init` in the examples/example-with-dynamodb-event directory. The bankrate registry has the same module, would it be possible to source from there?